feature/amazon/src/main/java/com/revenuecat/purchases/amazon/AmazonBilling.kt

```diff
@@ -352,4 +364,12 @@ internal class AmazonBilling constructor(
     private fun onPurchaseError(error: PurchasesError) {
         purchasesUpdatedListener?.onPurchasesFailedToUpdate(error)
     }
+
+    private fun checkObserverMode(): Boolean {
+        return if (observerMode) {
+            log(LogIntent.AMAZON_ERROR, AmazonStrings.ERROR_OBSERVER_MODE_NOT_SUPPORTED)
```

Comment:
wondering if this should be a warning, since we may call some of the unsupported methods ourselves...